src/components/molecules/NavBar/NavBar.scss

```diff
@@ -243,19 +201,74 @@ $border-radius: 28px;
 }
 
 .NavBar {
-  &__home-icon {
+  &__home--icon {
     cursor: pointer;
     font-size: 24px;
+    color: opaque-white(0.5);
 
     margin-right: $spacing--md;
-
-    &:hover {
-      font-size: 27px;
-      margin-right: $spacing--sm;
-    }
   }
 
   &__profile-popover {
     height: 100%;
   }
+
+  &__radio {
+    &--container {
+      padding: $spacing--lg $spacing--md;
+
+      & > iframe {
+        width: 100%;
+        border: none;
+      }
+    }
+  }
+
+  &__userAvatar {
+    width: $menu-item-size;
+    height: $menu-item-size;
+    margin-right: $spacing--sm;
+
+    img {
+      width: $menu-item-size;
+      height: $menu-item-size;
+    }
+  }
+
+  &__menu {
+    &--volume.mute {
+      opacity: 0.5;
+      box-shadow: 0 0 0 2px rgba($white, 1);
```

Comment:
`rgba($white,1)`  -> `$white` or `opaque-white(1)`

Reply:
[OOS] Would be good if the many `box-shadow` props through the project use few well defined presets (check `box-shadow--large` function for prior art)

Reply:
fixed rgba part 61d9df7